src/systems/webxr-dom-overlay.js

```diff
@@ -0,0 +1,21 @@
+var registerSystem = require('../core/system').registerSystem;
+
+/**
+ * WebXR DOM Overlay support.
+ */
+module.exports.System = registerSystem('webxr-dom-overlay', {
```

Comment:
maybe we can call it just `dom-overlay`? We can explain nuances in docs.

Reply:
Done, apart from the missing docs.